docs/source/interfaces/indirect/Inelastic Data Manipulation.rst

```diff
@@ -160,8 +145,9 @@ Input
   plot once a file has finished loading.
 
 Q Low, Q Width & Q High
-  Q binning parameters that are passed to the :ref:`SofQW <algm-SofQW>` algorithm. The low and high
-  values can be determined using the neighbouring contour plot.
+  Q binning parameters that are passed to the SofQW3 algorithm. The low and high
```

Comment:
can we have a link to the doc for the alg?

Reply:
I wasn't sure because it has been changed to `SofQW3` and I couldn't find a doc for that. Should I just keep it linked to `<algm-SofQW>`?

Reply:
@SpencerHowells which algorithm should it link to?

Reply:
@jhaigh0 @AnthonyLim23 SofQW3 is an alias for SofQWNormalisedPolygon so use that. Several reduction algorithms use SofQW3.